assets/queries/k8s/hpa_deployments_no_replicas/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "hpa_deployments_no_replicas",
+  "queryName": "HPA Deployments No Replicas",
```

Comment:
A more suitable name would be "HPA Targeted Deployments With Configured Replica Count"